app/views/layouts/_google_analytics.html.erb

```diff
@@ -1,11 +1,7 @@
-<% if ENV['GOOGLE_ANALYTICS_ID'].present? %>
 <!-- Global Site Tag (gtag.js) - Google Analytics -->
 <script async src="<%= "https://www.googletagmanager.com/gtag/js?id=#{ENV['GOOGLE_ANALYTICS_ID']}" %>"></script>
 <script>
   window.dataLayer = window.dataLayer || [];
   function gtag(){dataLayer.push(arguments)};
   gtag('js', new Date());
```

Comment:
in order to use the code above, I'm adding the line below:
`var GOOGLE_ANALYTICS_ID = '<%= ENV['GOOGLE_ANALYTICS_ID'] %>';`

Reply:
Not entirely sure what you mean.

Reply:
In projects based on standard js/jquery as opposed to angular/react I frequently find myself defining a select few global variables to re-use through the assets, e.g. in the application layout (or a partial therein) I'll put something like `var TRUSTBOOKS = {}` in js so that I can load up necessary ENV variables or other important (potentially dev/staging/production-specific configs), mainly just to keep them centralized (as opposed to in js.erb files). then I always know where to look, and js files stay as pure js files in case we want to do something unusual with the asset pipeline.

I don't know if that is relevant here, just something I've done in the past and continue to do now. Maybe it's something we should encourage in the template? Or maybe there is a best practice around this I don't know about 😁 

Anyway not super opinionated about these changes so I'll let you guys hash this out 🙏

Reply:
@roonglit If you wanna do me a favor it'd be great if you could add to this PR and maybe implement @hiattp's approach (which is pretty similar to yours). Maybe we can also standardize on a name, like e.g. `SETTINGS`.